<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librepcb::editor::BoardDesignRuleCheckDialog</class>
 <widget class="QDialog" name="librepcb::editor::BoardDesignRuleCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Design Rule Check</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="grpOptions">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Note: These values are not (yet) stored permanently in the project files, so they will be reset when closing the project.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbxRebuildPlanes">
          <property name="text">
           <string>Rebuild planes before running checks</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbxClearanceCopperCopper">
          <property name="text">
           <string>Clearance Copper to Copper:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtClearanceCopperCopper" native="true"/>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbxClearanceCopperBoard">
          <property name="text">
           <string>Clearance Copper to Board Edge:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtClearanceCopperBoard" native="true"/>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="cbxClearanceCopperNpth">
          <property name="text">
           <string>Clearance Copper to NPTH Drills:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtClearanceCopperNpth" native="true"/>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="cbxMinCopperWidth">
          <property name="text">
           <string>Minimum Copper Width:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtMinCopperWidth" native="true"/>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="cbxMinPthAnnularRing">
          <property name="text">
           <string>Minimum PTH Annular Ring:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtMinPthAnnularRing" native="true"/>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="cbxMinNpthDrillDiameter">
          <property name="text">
           <string>Minimum NPTH Drill Diameter:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtMinNpthDrillDiameter" native="true"/>
        </item>
        <item row="9" column="0">
         <widget class="QCheckBox" name="cbxMinPthDrillDiameter">
          <property name="text">
           <string>Minimum PTH Drill Diameter:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtMinPthDrillDiameter" native="true"/>
        </item>
        <item row="11" column="0">
         <widget class="QCheckBox" name="cbxWarnNpthSlots">
          <property name="text">
           <string>Warn About NPTH Slots:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="cbxWarnNpthSlotsConfig">
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QCheckBox" name="cbxCourtyardOffset">
          <property name="text">
           <string>Additional Courtyard Offset:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="librepcb::editor::LengthEdit" name="edtCourtyardOffset" native="true"/>
        </item>
        <item row="14" column="0">
         <widget class="QCheckBox" name="cbxMissingConnections">
          <property name="text">
           <string>Check for missing connections</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QPushButton" name="btnSelectAll">
          <property name="text">
           <string>Select All/None</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="0" colspan="2">
         <widget class="QProgressBar" name="prgProgress">
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="cbxMinNpthSlotWidth">
          <property name="text">
           <string>Minimum NPTH Slot Width:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtMinNpthSlotWidth" native="true"/>
        </item>
        <item row="10" column="0">
         <widget class="QCheckBox" name="cbxMinPthSlotWidth">
          <property name="text">
           <string>Minimum PTH Slot Width:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="librepcb::editor::UnsignedLengthEdit" name="edtMinPthSlotWidth" native="true"/>
        </item>
        <item row="12" column="0">
         <widget class="QCheckBox" name="cbxWarnPthSlots">
          <property name="text">
           <string>Warn About PTH Slots:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QComboBox" name="cbxWarnPthSlotsConfig">
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Messages</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="lstMessages"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>librepcb::editor::LengthEdit</class>
   <extends>QWidget</extends>
   <header location="global">librepcb/editor/widgets/lengthedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>librepcb::editor::UnsignedLengthEdit</class>
   <extends>QWidget</extends>
   <header location="global">librepcb/editor/widgets/unsignedlengthedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbxRebuildPlanes</tabstop>
  <tabstop>cbxClearanceCopperCopper</tabstop>
  <tabstop>edtClearanceCopperCopper</tabstop>
  <tabstop>cbxClearanceCopperBoard</tabstop>
  <tabstop>edtClearanceCopperBoard</tabstop>
  <tabstop>cbxClearanceCopperNpth</tabstop>
  <tabstop>edtClearanceCopperNpth</tabstop>
  <tabstop>cbxMinCopperWidth</tabstop>
  <tabstop>edtMinCopperWidth</tabstop>
  <tabstop>cbxMinPthAnnularRing</tabstop>
  <tabstop>edtMinPthAnnularRing</tabstop>
  <tabstop>cbxMinNpthDrillDiameter</tabstop>
  <tabstop>edtMinNpthDrillDiameter</tabstop>
  <tabstop>cbxMinNpthSlotWidth</tabstop>
  <tabstop>edtMinNpthSlotWidth</tabstop>
  <tabstop>cbxMinPthDrillDiameter</tabstop>
  <tabstop>edtMinPthDrillDiameter</tabstop>
  <tabstop>cbxMinPthSlotWidth</tabstop>
  <tabstop>edtMinPthSlotWidth</tabstop>
  <tabstop>cbxWarnNpthSlots</tabstop>
  <tabstop>cbxWarnNpthSlotsConfig</tabstop>
  <tabstop>cbxWarnPthSlots</tabstop>
  <tabstop>cbxWarnPthSlotsConfig</tabstop>
  <tabstop>cbxCourtyardOffset</tabstop>
  <tabstop>edtCourtyardOffset</tabstop>
  <tabstop>cbxMissingConnections</tabstop>
  <tabstop>btnSelectAll</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
